### PR TITLE
[vividus-plugin-web-app] Refactor opera to reuse chrome driver

### LINKED ISSUE #2896: Refactor opera to reuse chrome driver
https://github.com/SeleniumHQ/selenium/blob/d0b95a04a5aaaa88bfcd10705b8231ba44e5b774/java/src/org/openqa/selenium/opera/OperaDriver.java#L42-L62

### PR DESCRIPTION
**Pull request by VolhaHurynovich**
Closes #2896

### CODE REVIEW

**Comment by valfirst** [comment]
Please update the docs on how to use Opera

**Comment by VolhaHurynovich** [comment]
> Please update the docs on how to use Opera

There is nothing special (It works without property` web.driver.OPERA.binary-path=C:/....Programs/Opera/opera.exe`)

**Comment by valfirst** [comment]
> It works without property` web.driver.OPERA.binary-path=C:/....Programs/Opera/opera.exe`

How is that possible?

**Comment by VolhaHurynovich** [comment]
> > It works without property` web.driver.OPERA.binary-path=C:/....Programs/Opera/opera.exe`
> 
> How is that possible?

after I deleted this property and run `gradlew cleanEclipse eclipse`, it was working. Did I clear the cache correctly or not?

**Comment by VolhaHurynovich** [comment]
added Note for using Opera in docs

File: docs/modules/plugins/pages/plugin-web-app.adoc [hunk 1]
```diff
@@ -201,6 +201,15 @@ a|The command line (CLI) arguments to use when starting browser.
 
 |===
 
+NOTE: For using Opera browser it is mandatory property:
```
Comment (ikalinin1) [inline docs/modules/plugins/pages/plugin-web-app.adoc:204]: Is this the right place for this? Maybe instead add a description for the web/desktop/opera profile and include this in the description

Comment (VolhaHurynovich) [reply docs/modules/plugins/pages/plugin-web-app.adoc]: moved to `Profiles`

**Review by valfirst — APPROVED** [review]

**Review by ikalinin1 — APPROVED** [review]

**Review by uarlouski — APPROVED** [review]

**Pull request merged**